docs/v2-release-notes.md

```diff
@@ -3,3 +3,21 @@
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
 ## Jun 24, 2024
+
+Timeplus Enterprise v2 is now available for cloud or self-hosted deployment. [See installation options](timeplus.com/product#selfhosted) for self-hosting. 
+
+### Timeplus Core Engine (Proton v1.5.11):
```

Comment:
`Timeplus Core Engine` is now a bit misleading, since the Timeplus Enterprise ships timeplusd. Let's just call this `Timeplus Proton 1.5.11`